client/a8c-for-agencies/sections/migrations/consolidated-commissions/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { Card } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+
+import './style.scss';
+
+export default function MigrationsConsolidatedCommissions() {
+	const translate = useTranslate();
+
+	const getCurrentQuarter = () => {
+		const date = new Date();
+		const quarter = Math.floor( ( date.getMonth() + 3 ) / 3 );
+		switch ( quarter ) {
+			case 1:
+				return 'Q1';
+			case 2:
+				return 'Q2';
+			case 3:
+				return 'Q3';
+			case 4:
+				return 'Q4';
+			default:
+				return '';
+		}
+	};
```

Comment:
This function could be simplified to:

```
const getCurrentQuarter = () => {
    const month = new Date().getMonth();
    const quarter = Math.floor(month / 3) + 1;
    return `Q${quarter}`;
};
```

Reply:
Thanks for the review, @vitozev!

I have updated the logic: https://github.com/Automattic/wp-calypso/pull/96225/commits/5a2fb6ddd3c80338f02ba76635e14ac9a364d45a

Could you please take another look?